woocommerce/woocommerce-gutenberg-products-block
# Single Product Block Edit Mode

_#2399 is a mammoth PR so I'm breaking out some fixes and changes to review separately._

This PR adds the Edit View to the Single Product Block (the product selection part). _This does not include the inner block editor._

While I was adding the block preview, I cleaned up some other resource images to reduce bundle size.

### Screenshots

![Edit Page ‹ Local Testing — WordPress 2020-05-27 16-47-47](https://user-images.githubusercontent.com/90977/83043280-94031a00-a03a-11ea-9cd8-1e7835a756f9.png)

### How to test the changes in this Pull Request:

1. Use the inserted and find single product block
2. Ensure there is a skeleton like block preview
3. Insert the block. Test the product selection and click done. The block should go blank (expected at this point).
4. Check the inspector. When you're in the selected state, you should see a "Product" Section. This should contain an instance of the product selector.
